django/core/checks/urls.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from . import Tags, Warning, register
+
+W001 = Warning(
+    "In your url patterns, you have used include with a regex containing a $. "
```

Comment:
Good idea, I've simplified the text. I went for "One of your url patterns..." because it's simpler than "Your url patterns have a patten...".